_members/delatorre-clara.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Clara De La Torre
+image: images/delatorre-clara.jpg
+role: phd
+links:
+  email: czd0089@auburn.edu
+education:
+  - Texas A&M, B.S. in Chemical Engineering
+---
+
+Clara joined the lab in 2023.
```

Comment:
Can you rewrap this text? Each sentence doesn't have to start on a new line, and we try to keep lines < 80 characters. If you're using VSCode, there is a rewrap tool.